xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -229,9 +229,11 @@
         <ReturnType>System.Collections.ObjectModel.Collection&lt;System.String&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a collection of command-line arguments to use when starting the application.</summary>
+        <value>A collection of string values.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[If you use this property to set command-line arguments, <xref:System.Diagnostics.ProcessStartInfo.Arguments*> must be set to an empty string.]]></format>
```

Comment:
nit: 

Can you encode * as %2A or maybe even remove it?
I think it doesn't cause issues here because it's a mixture of Markdown and ECMAXML, but we do have to encode when it's pure MarkDown, so it would be better not to worry about this when we migrate. 

Also, because this property only has a getter, we probably need to show to you set the values, no?